lib/ansible/runner/credential/openstack_credential.rb

```diff
@@ -0,0 +1,44 @@
+module Ansible
+  class Runner
+    class OpenstackCredential < Credential
+      def self.auth_type
+        "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::OpenstackCredential"
+      end
+
+      # Modeled off of openstack injectors for awx:
+      #
+      #   https://github.com/ansible/awx/blob/1242ee2b/awx/main/models/credential/injectors.py#L70-L96
+      #
+      def env_vars
+        write_credentials_file
+        { "OS_CLIENT_CONFIG_FILE" => os_credentials_file }
+      end
+
+      def write_credentials_file
+        openstack_data = {
+          "clouds" => {
+            "devstack" => {
+              "verify" => false, # TODO:  NickL:  Should we implement this?
```

Comment:
I'm not sure if we were passing anything before when creating the credential on the AWX side, but the default for this there is `true`...

Reply:
I agree, though I figured organizations would more likely running into issues because they didn't setup certs for hosts and this would be a friendlier default (though, very much less secure).

I realized it was the default, but the comment was specifically there because we have no way being able to change it through other means currently (unlike tower).

Reply:
Indeed, I'm fine with this as is.